README.md

```diff
@@ -8,9 +8,9 @@ torchao is a PyTorch native library for optimizing your models using lower preci
 
 The library provides
 1. Support for lower precision [dtypes](./torchao/dtypes) such as nf4, uint4 that are torch.compile friendly
-2. Quantization [algorithms](./torchao/quantization) such as dynamic quant, smoothquant, GPTQ that run on CPU/GPU and Mobile.
-3. Sparsity [algorithms](./torchao/sparsity) such as Wanda that help improve accuracy of sparse networks
-4. Integration with other PyTorch native libraries like torchtune and ExecuTorch
+2. [Quantization algorithms](./torchao/quantization) such as dynamic quant, smoothquant, GPTQ that run on CPU/GPU and Mobile.
```

Comment:
@supriyar I was hoping that people can just find them here, or do you feel we want to make it more explicit?

Reply:
I do agree that a getting started on the main page would be important to keep. So if GPTQ is the algorithm we feel most people would be interested in let's just link that